pkg/services/postgresql/database_provision.go

```diff
@@ -81,30 +79,29 @@ func (d *dbOnlyManager) deployARMTemplate(
 	return dt, instance.SecureDetails, nil
 }
 
-func (d *dbOnlyManager) setupDatabase(
+func (d *databaseManager) setupDatabase(
```

Comment:
I've dropped "only" wherever I could because I think the role of these managers is clear without "only" in their name. I _did_ keep "only" in relevant catalog descriptions, since that is seen by the end user.